### ISSUE TITLE
{Match.Time:HH:mm} on {Match.Time:ddd dd/MM}

### ISSUE THREAD

**Issue by paulb2011** [scottrippey/SmartFormat.NET#63]
After upgrading to version 1.6.1, formatting the time with a colon no longer works:

```
    class Program
    {
        class Example
        {
            public Example()
            {
                Time = DateTime.UtcNow;
            }

            public DateTime Time { get; set; }
        }

        static void Main( string[] args )
        {
            var template = "Time: {Time:HH:mm} Date: {Time:ddd dd/MM}.";

            var text = Smart.Format( template, new Example() );

            // outputs: Time:  Date: Sun 21/06.

            Console.WriteLine( text );

            Console.ReadLine();
        }
    }

```

**Comment by scottrippey** [comment]
Should be solved with #65

**Comment by axunonb** [comment]
Will be closed in upcoming version 1.7.x shortly:
When the parser comes across a formatter name, this does not exist in the registered extensions, it will treat this part as a format, which will be processed by DefaultFormatter.

**Comment by axunonb** [comment]
Updated version committed, where test with the issues mentioned have passed: https://github.com/scottrippey/SmartFormat.NET/commit/46bbae26b6c34582887b60fc0488cd134514fde7
